src/scss/pages/_global.scss

```diff
@@ -214,6 +214,9 @@ h2 {
       }
       p {
         margin-bottom:0;
+        & + .highlight {
+          margin-top:10px;
```

Comment:
Could you please add a space: `margin-top: 10px;`

Reply:
👌